### PR TITLE
Added acados

### PR DESCRIPTION
**Pull request by pariterre**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pariterre** [comment]
@conda-forge-admin, please rerender

File: recipes/acados/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  build:
+    - pkg-config
+    - cmake
+    - ninja
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - {{ pin_compatible('numpy') }}
```
Comment (marcelotrevisani) [inline recipes/acados/meta.yaml:30]: In host it should be just ``numpy``, the pin_compatible in run is correct :)
```suggestion
    - numpy
```

File: recipes/acados/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - acados
+
+about:
+  home: https://github.com/acados/acados
+  license: BSD License
+  license_file: LICENSE.txt
```
Comment (marcelotrevisani) [inline recipes/acados/meta.yaml:42]:
```suggestion
  license_file: LICENSE
```

File: recipes/acados/build.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+pushd build
+
+cmake $SRC_DIR \
+  -GNinja \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX}\
+  -DACADOS_INSTALL_DIR=${PREFIX}\
+  -BLASFEO_TARGET="GENERIC" \
+  -DACADOS_WITH_QPOASES=ON \
+  -DACADOS_PYTHON=ON \
+  -DPYTHON_PREFIX=${SP_DIR} \
```
Comment (marcelotrevisani) [inline recipes/acados/build.sh:11]:
```suggestion
  -DPYTHON_PREFIX=${SP_DIR} \
  -DCMAKE_BUILD_TYPE=Release \
```

**Comment by pariterre** [comment]
@marcelotrevisani 
Thanks for reaching out. This PR won't work for quite a for time though, I am afraid. The problem comes from how the external packages are included. This is incompatible so far with the philosophy of conda-forge (that is packaging everything at build time instead of relying on each package being build separately and hosted on conda-forge). I have to decide if I have the time to create a conda-forge package for each dependency and modify the CMakeList accordingly or just drop the idea of having acados on conda-forge

@FreyJo any thoughts about this?

**Pull request closed without merging**